.circleci/config.yml

```diff
@@ -37,6 +37,8 @@ jobs:
       - run:
           name: Test and Code Coverage
           command: npm run test
+      - codecov/upload:
+          flags: unittests
```

Comment:
Found the reason for the codecov discrepancy: needs to upload both coverage outputs